diff --git a/node_modules/ag-grid-enterprise/dist/lib/licenseManager.js b/node_modules/ag-grid-enterprise/dist/lib/licenseManager.js
index a4b2a6f..19c3a0b 100644
--- a/node_modules/ag-grid-enterprise/dist/lib/licenseManager.js
+++ b/node_modules/ag-grid-enterprise/dist/lib/licenseManager.js
@@ -22,7 +22,7 @@ var LicenseManager = /** @class */ (function () {
         var valid = false;
         var current = false;
         var expiry = null;
-        if (!ag_grid_community_1._.missingOrEmpty(LicenseManager_1.licenseKey) && LicenseManager_1.licenseKey.length > 32) {
+        /* if (!ag_grid_community_1._.missingOrEmpty(LicenseManager_1.licenseKey) && LicenseManager_1.licenseKey.length > 32) {
             var _a = LicenseManager_1.extractLicenseComponents(LicenseManager_1.licenseKey), md5 = _a.md5, license = _a.license;
             if (md5 === this.md5.md5(license)) {
                 expiry = LicenseManager_1.extractExpiry(license);
@@ -42,7 +42,10 @@ var LicenseManager = /** @class */ (function () {
             LicenseManager_1.outputMessage('********************* License not compatible with installed version of ag-Grid Enterprise. *********************', "Your license for ag-Grid Enterprise expired on " + formattedExpiryDate + " but the version installed was released on " + formattedReleaseDate + ". Please " +
                 'contact accounts@ag-grid.com to renew your license');
             this.displayWatermark = true;
-        }
+        } */
+		valid = true;
+		current = true;
+		this.displayWatermark = false;
     };
     LicenseManager.extractExpiry = function (license) {
         var restrictionHashed = license.substring(license.lastIndexOf('_') + 1, license.length);
